itou/job_applications/models.py

```diff
@@ -323,6 +323,12 @@ class Meta:
         verbose_name_plural = _("Candidatures")
         ordering = ["-created_at"]
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        for state, desc in JobApplicationWorkflow.STATE_CHOICES:
```

Comment:
`for state, _ in ...` (astuce!)

Reply:
Dommage d'avoir déjà importé `gettext_lazy` avec l'alias `_`
Je propose de mettre une variable `_desc` (underscore = OSEF mais on sait quand même ce que c'est)